packages/fxa-auth-server/lib/senders/templates/subscriptionCancellation.html

```diff
@@ -7,7 +7,7 @@
 {{/inline}}
 
 {{#*inline "content"}}
-  {{t "We've cancelled your %(productName)s subscription. Your final payment of $%(invoiceTotal)s was paid on %(invoiceDateOnly)s. Your service will continue until the end of your current billing period, which is %(serviceLastActiveDateOnly)s." }}
+  {{t "We've cancelled your %(productName)s subscription. Your final payment of %(invoiceTotal)s was paid on %(invoiceDateOnly)s. Your service will continue until the end of your current billing period, which is %(serviceLastActiveDateOnly)s." }}
```

Comment:
I know this isn't from your PR, but I'm 😦 that we're mixing curly and straight quotes across email copy

Reply:
We are? Hmm... I think this copy was mainly pasted in from a word doc, but I could have biffed it up in a few spots.

Reply:
I only caught two instances of curlies, [here](https://github.com/mozilla/fxa/pull/5602/files#diff-dd54afbfc7b84fc299c33974b61a8664R10) and [here](https://github.com/mozilla/fxa/pull/5602/files#diff-0f3aeadb92255befe17def7a6f5d06c2R5)

Reply:
Hmm, maybe I'll just straighten those out while I'm in here.